src/components/RadioGroup/examples/default.jsx

```diff
@@ -8,7 +8,7 @@ const style = {
 export default React.createClass({
 	render() {
 		return (
-			<RadioGroup>
+			<RadioGroup name='name'>
```

Comment:
This kind of changes the nature of the example a bit. One main feature is that you don't have to provide an arbitrary name. but now all the examples include it so that the tests will be consistent.